src/src/PluginStructs/P176_data_struct.cpp

```diff
@@ -151,6 +151,23 @@ bool P176_data_struct::plugin_get_config_value(struct EventStruct *event,
     const String key = parseString(string, 1, '.'); // Decimal point as separator, by convention
     VictronValue value;
 
+    # if P176_HANDLE_CHECKSUM
+
+    if (equals(key, F("successcount"))) {
+      string  = _successCounter;
+      success = true;
+    } else
+    if (equals(key, F("ischanged"))) {
```

Comment:
Maybe better term would be "updated" or "newvalue" ?
Or else "changed" without the "is..."

And I think these 3 should be made available for all plugins, however this can be done later in another issue :)